rskj-core/src/main/java/org/ethereum/core/Block.java

```diff
@@ -455,6 +458,8 @@ public String toFlatString() {
     }
 
     private void parseTxs(RLPList txTransactions) {
+        ArrayList<Transaction> builder = new ArrayList<>();
```

Comment:
It's recommended to use an interface whenever you can, in this case _builder_ could be  `List<Transaction>`. Also I think the name of the variable could be improved, what about _parsedTransactions_?